pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go

```diff
@@ -226,3 +384,45 @@ func deserializePipelineRunResources(meta *metav1.ObjectMeta, spec *PipelineRunS
 	}
 	return nil
 }
+
+// convertTaskRunsToChildReference handles the deprecated TaskRuns field PipelineRunTaskRunStatus.
+// The apiViersion is set to 'v1beta1' as default, it could be changed if v1 is swapped as stored.
```

Comment:
```suggestion
// The apiVersion is set to 'v1beta1' because we treat all TaskRuns within a PipelineRun
// as having the same API version as the PipelineRun.
```